homeassistant/components/roborock/services.yaml

```diff
@@ -0,0 +1,42 @@
+vacuum_set_mop_mode:
```

Comment:
Why isn't this a select entity?

Reply:
Good call - Wasn't something I thought of at first but definitely a better solution - I have made the change.